test/test_cpp20/test_cpp20.vcxproj

```diff
@@ -39,9 +39,12 @@
     <ProjectGuid>{5FF6CD6C-515A-4D55-97B6-62AD9BCB77EA}</ProjectGuid>
     <RootNamespace>unittests</RootNamespace>
     <ProjectName>test_cpp20</ProjectName>
-    <CppWinRTLanguageStandard>latest</CppWinRTLanguageStandard>
+    <CppWinRTLanguageStandard>stdcpp20</CppWinRTLanguageStandard>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
+  <PropertyGroup Label="Configuration">
+    <PlatformToolset>v143</PlatformToolset>
```

Comment:
Instead of setting this here, we should simply delete `test/Directory.Build.props`. The issue with catch and VS 2022 appears to be fixed.